src/components/Climate/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import temp from './temp.svg';
+import './style.css';
+
+
+const Climate = ({temperature, humidity}) => {
+
+    const [showTemp, setShowTemp] = useState(temperature);
+
+    const changeHouseTemp = (e) => setShowTemp(showTemp => showTemp + (e.target.innerText === '+' ? 1 : -1));
```

Comment:
tohle se sice dá, ale reálně bych to nepoužíval. Spoléhat na nějaký text elementu je nebezpečný už jen proto, že když ho někdo změní (třeba přidá <span>+</span>) nebo když máš vícejazyčnou aplikaci.... Tj je to strašně náchylný k bugům. Tj radši to udělat podobně jako jsi měla v Blinds - onOpen a onClose